conda_forge_tick/audit.py

```diff
@@ -54,6 +56,8 @@
     "versioneer",
     # not a real dep
     "cross-python",
+    "c_compiler_stub",
+    "cxx_compiler_stub",
```

Comment:
you might as well put the rest of them in here, especially fortran_compiler_stub and cuda_compiler_stub

Reply:
Do we have a list of all the stubs?

Reply:
nevermind I got it